mne/channels/channels.py

```diff
@@ -502,8 +502,9 @@ def plot_sensors(self, kind='topomap', ch_type=None, title=None,
         title : str | None
             Title for the figure. If None (default), equals to ``'Sensor
             positions (%s)' % ch_type``.
-        show_names : bool
-            Whether to display all channel names. Defaults to False.
+        show_names : bool | array of str
```

Comment:
hmm ... that makes sense. In that case, this LGTM. Any test to update? Why is codecov complaining? Also you have to update `whats_new` I suppose